src/main/java/alfred/action/TaskList.java

```diff
@@ -0,0 +1,132 @@
+package alfred.action;
+
+import alfred.task.Task;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+/**
+ *  Encapsulates a list containing Task objects. <code>TaskList</code> contains
+ *  <code>ArrayList<Task></code>
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Creates a new TaskList object.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Adds task object into TaskList. Only a Task object can be passed into
+     * the method, which will be stored in tasks.
+     *
+     * @param newTask the new Task object to be added
+     *
```

Comment:
Similar to above, perhaps removing this extra line for the java doc would be more synchronised with the coding standards?